runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -384,6 +384,16 @@ public void onGameTick(final GameTick event)
 			}
 		}
 
+		if (clue instanceof FairyRingClue)
```

Comment:
I would probably merge this with CoordinateClue if (e.g instanceof Fairy || instanceof Coordinate) and fix coordinate clue to store the Inventory in InventoryItems instead of EquippedItems